src/Symfony/Component/Routing/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 7.2
 ---
 
+ * Add aliases in the Route attribute
```

Comment:
New changelog entries should be appended, not prepended. Also 7.2 is in feature freeze so you may want to create the 7.3 entry directly